app/views/common_parameters/_puppetclass_parameter.html.erb

```diff
@@ -5,10 +5,10 @@
         <td class="col-md-3">
           <%= f.hidden_field :lookup_key_id, :'data-property' => 'lookup_key_id' %>
           <% puppetclass = f.object.lookup_key ? f.object.lookup_key.param_class : nil %>
-          <%= text_field_tag '', (puppetclass.name rescue ''), :class => 'form-control', :'data-property' => 'class', :disabled => true, :title => _('Puppetclass') %>
+          <%= content_tag :span, "#{puppetclass.name rescue ''}", :'data-property' => 'class' %>
```

Comment:
Any need for the string interpolation on these two changes?  You should be able to remove it.